streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -168,20 +165,26 @@
                                    String name);
 
     /**
-     * Aggregate values of this table by the selected key.
+     * Aggregate updating values of this stream by the selected key into a new instance of {@link KTable}.
```

Comment:
General comment:  Keep the current docs as is, but I think for such a behemoth of a function (look at the number of params!) we should add code examples to the docstring in the future.  Most users will struggle to write their first `aggregate` without seeing a simple example first.